src/Raven.Server/Commercial/LicenseValidator.cs

```diff
@@ -39,7 +39,7 @@ private static byte[] GetBytesFromBase64String(string str)
             return Convert.FromBase64String(str);
         }
 
-        public static Dictionary<string, object> Validate(License licenseKey, RSAParameters rsAParameters)
+        public static Dictionary<string, object> Validate(License licenseKey, RSAParameters rsaParameters)
```

Comment:
Shouldn't be using `sodium` here to do this? That would be simpler to handle, I think.